snapcraft.yaml

```diff
@@ -26,18 +29,25 @@ apps:
       - audio-record
       - bluez
       - camera
+      - content
+      - cups
       - cups-control
       - home
       - joystick
       - mount-observe
       - network
+      - network-bind
```

Comment:
It would be nice to know if this is needed for binding *inside* the container, where I presume we run the ipfs and tor daemons, or only binding to an interface available *outside* the container.

Anyway, per slack discussion, we should default to open so the snap works as well as the .deb packages we publish.